README.md

```diff
@@ -5,17 +5,31 @@ filexp
 
 ## Installation
 
+There are two flavors of filexp that you can install:
+1. Fully featured, including various commands for interacting with the FVM and FEVM
+2. A more limited version that handles most commands (currently everything but `fevm-exec` and `fevm-daemon`)
+   - This version is the only one that works on Windows
+
+### Fully featured
+
 Install Go and run:
 
 ```
 git clone https://github.com/aschmahmann/filexp
 cd filexp
 git submodule update --init
 make -C extern/filecoin-ffi
-go install
+go install -tags fvm
 ```
 
-Note: Currently does not work on Windows
+### Limited
+
+Install Go and run:
+```
+git clone https://github.com/aschmahmann/filexp
+cd filexp
+go install
```

Comment:
Note: it'd be great to also enable `go install github.com/aschmahmann@latest`. Currently that won't work due to replace directives.

One is from the go-ipld-hamt fork, which seems easily fixable. The other is from filecoin-ffi which can't be fixed until they've made it into a proper module (IIUC some ongoing work). Haven't experimented to find out if the replace will get ignored if it's not needed which would mean we could go ahead with just the hamt fork